core/chaincode/shim/crypto/ac/abac.go

```diff
@@ -0,0 +1,145 @@
+/*
```

Comment:
should use https://github.com/hyperledger/fabric/blob/master/docs/dev-setup/headers.txt replacing "[COPYRIGHT OWNER]" with "IBM Corp. 2016"